testing/trino-product-tests/src/main/java/io/trino/tests/product/utils/DatabricksUnityCatalogUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.utils;
+
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public final class DatabricksUnityCatalogUtils
+{
+    private DatabricksUnityCatalogUtils()
+    {
+    }
+
+    public static String getTestCatalogName()
+    {
+        return requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_NAME"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_NAME");
+    }
+
+    public static String getExternalLocationName()
+    {
+        return requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_EXT_LOC"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_EXT_LOC");
+    }
+
+    public static String getS3BucketName()
+    {
+        return requireNonNull(System.getenv("S3_BUCKET"), "Environment variable not set: S3_BUCKET");
+    }
```

Comment:
Inline them in the `setup` method of the classes which need them.

https://github.com/trinodb/trino/blob/470f666be5a1cec41891963f842a7f3e6a808f1d/testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/BaseTestDeltaLakeS3Storage.java#L26-L30

Reply:
Is there a commit missing from your submission? I don't see this comment addressed.